README.md

```diff
@@ -1,114 +1,80 @@
+constraints that were violated.
 
-Type the number of employees to be considered and hit `Enter`. Note that the
-CQM solver can take up to 5000 variables, or employees for this problem.
+## Introducing the Demo
 
-A second prompt for the number of shifts will appear:
+When the demo opens in the web browser, there are a few options on the left and
+a chart of employee availability in the center.
 
-```Enter number of shifts:```
+![Screen Image](assets/demo_image.png)
 
-Type the number of shifts and hit `Enter`. Note that we should have at least as
-many employees as there are shifts.
+### Employee Availability
 
-Once these values have been entered, the program will randomly generate employee
-preferences for the N shifts from most preferred (0) to least preferred (N-1). A
-CQM is constructed (see below for details) and the problem is run using
-`LeapHybridCQMSampler`.
+The employee availability chart shows employee shift preferences and unavailable
+days (PTO). Preferred shifts are in teal and marked with a 'P', while
+unavailable shifts are in organe and marked with an 'X'.
 
-Once the problem has run, two images are created. First, `employee_schedule.png`
-illustrates the employee preference matrix alongside the schedule built.
-Second, `schedule_statistics.png` shows how many employees are scheduled for
-each shift, alongside a bar chart showing the employees' average preferences
-for the shifts for which they have been scheduled.
+In the chart, there are three different types of employees.
 
-At the end of the program's run, two statistics are printed to the
-command-line. Schedule score provides the energy value of the best solution
-found. Average happiness is the average of the employee preference values for
-the shifts that they are scheduled. A smaller average happiness score is
-better, and a score of 0.0 is a perfect score - everyone received their first
-choice shift.
+- Managers: These are employees with 'Mgr' at the end of their name.
+- Employees: These are employees with no tag at the end of their name.
+- Trainees: These are employees with 'Tr' at the end of their name. The trainee
+  has the same name as their trainer. The trainee can **only** be scheduled to
+  work on a shift that their trainer is also scheduled to work.
 
-To run the DQM demo, type the command:
+The chart displays employee preferences and availability for this month. It will
+always display the current month, with one column for each day in this current
+month.
 
-```python scheduler.py```
+### Options
 
-Similar commands and output follow.
+Always available are three options.
 
-## Building the Quadratic Model
+- Number of employees: Pretty self-explanatory. We will always have 2 managers
+  and 1 trainee.
+- Random seed: Optional; included for consistency when giving live demos.
+- Autofill scenario: Auto-populates all settings to provide scenarios of varying
+  sizes that will produce feasible solutions.
```

Comment:
```suggestion
```

Should be moved to under ###Inputs per the updated [example template](https://github.com/dwave-examples/template)

I edited the phrasing in a separate suggested change under ###Inputs